theurgy.lic

```diff
@@ -40,11 +40,51 @@ def comm_tasmine
   put_in_container($settings.water_holder)
 end
 
+def comm_kert
+  return unless DRStats.circle > 8 && DRSkill.getxp('Theurgy') <= 31
+  case DRC.bput("get oil from #{$settings.theurgy_supply_container}", 'holy oil', 'some oil', 'I could not find', 'What were you referring to')
+  when 'I could not find', 'What were you referring to'
+    return
+  when 'holy oil'
+  when 'some oil'
+    if 'interrupt your research' == DRC.bput('prep bless', 'Bless spell', 'interrupt your research')
```

Comment:
I'd like to see this guy pulled out into a pre-emptive function that checks the result of a `research status` to give the all clear for casting or not. We'd probably need to improve crossing training research flags so it notices the cancel and restarts it when it gets a chance.